### PR TITLE
fixed link to node-oracledb

### PR DESCRIPTION
**Pull request by rashmihunt**
Fixed link to node-oracledb  in Readme

@raymondfeng PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ For full documentation, see the official [StrongLoop  Documentation](http://docs
 
 ## Installation
 
-To simplify the installation of [node-oracle](https://github.com/strongloop/node-oracle) module and Oracle instant clients,
+To simplify the installation of [node-oracledb](https://github.com/oracle/node-oracledb) module and Oracle instant clients,
 we introduce [loopback-oracle-installer](https://github.com/strongloop/loopback-oracle-installer) as a dependency which installs
-and configures node-oracle upon `npm install`.
+and configures node-oracledb(oracledb) upon `npm install`.
```
Comment (superkhau) [inline README.md]: space between b(

File: README.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ For full documentation, see the official [StrongLoop  Documentation](http://docs
...
 
-Please note `config.oracleUrl` is the property to define the base URL to download the corresponding node-oracle bundle for the local
+Please note `config.oracleUrl` is the property to define the base URL to download the corresponding node-oracle(oracledb) bundle for the local
```
Comment (superkhau) [inline README.md]: here too

File: README.md (hunk 1 above)
Comment (rashmihunt) [inline README.md]: fixed

**Comment by rashmihunt** [comment]
@slnode test please

**Pull request merged**